alibi/api/interfaces.py

```diff
@@ -41,6 +42,25 @@ def __attrs_post_init__(self):
     def explain(self, X: Any) -> "Explanation":
         pass
 
+    def _update_metadata(self, data_dict: dict, params: bool = False) -> None:
+        """
+        This method updates the metadata of the explainer using the data from the `data_dict`. If the params option
```

Comment:
Should just start with a verb, like the other docstrings here: "Updates  the metatata..."